services/paginateResponse.js

```diff
@@ -0,0 +1,153 @@
+const proverb = require('../models/proverb')
+
+function addNextPreviousTotalObj (page, limit, arrLength) {
+  const startIndex = (page - 1) * limit
+  const endIndex = page * limit
+
+  const results = {}
+
+  results.total_pages = Math.ceil(arrLength / limit)
+
+  if (endIndex < arrLength) {
+    results.next = {
+      page: page + 1,
+      limit: limit
```

Comment:
```suggestion
      limit,
```
You don't need to add the value of an object when the variable name is the same as the key.